docs/http/http-spans.md

```diff
@@ -240,6 +242,21 @@ The following attributes can be important for making sampling decisions and SHOU
 | `unix` | Unix domain socket | ![Stable](https://img.shields.io/badge/-stable-lightgreen) |
 <!-- endsemconv -->
 
+### HTTP Client Experimental Attributes
+
+**Status**: [Experimental][DocumentStatus]
+
+Instrumentations MAY allow users to enable additional experimental attributes. The following attributes are define for HTTP Client spans
```

Comment:
(maybe not needed, and then don't need to decide whether to end in `.` or `:` 😂)
```suggestion
Instrumentations MAY allow users to enable additional experimental attributes.
```